obbi/HA-active-passive-deployment/Chart.yaml

```diff
@@ -0,0 +1,15 @@
+# ----------------------------------------------------------------------------------------
+#
+# Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
+#
+# This software is the property of WSO2 LLC. and its suppliers, if any.
+# Dissemination of any information or reproduction of any material contained
+# herein in any form is strictly forbidden, unless permitted by WSO2 expressly.
+# You may not alter or remove any copyright or other notice from copies of this content.
+#
+# ----------------------------------------------------------------------------------------
+
+apiVersion: v1
+name: obbi
+description: A Helm chart for the deployment of WSO2 Open Banking Business Intelligence module (HA), This chart depends on wso2 obiam and wso2 obam.
```

Comment:
Shall we remove `HA` from the folder name?  `active-passive-deployment`

Reply:
Done with ee09195137401f5ea9b3fc57f3cc4f70e8a48893